datadog/resource_datadog_dashboard_json.go

```diff
@@ -112,6 +143,12 @@ func resourceDatadogDashboardJSONCreate(ctx context.Context, d *schema.ResourceD
 	}
 	d.SetId(id.(string))
 
+	layoutType, ok := respMap["layout_type"]
+	if !ok {
+		return diag.FromErr(errors.New("error retrieving layout_type from response"))
+	}
+	updateDashboardJSONLists(d, providerConf, id.(string), layoutType.(string))
```

Comment:
Hmm this is tricky I think in case of create. We might need to add some retries here, make sure the dashboard is fully replicated in the backend, otherwise I believe the call to add to a list will fail.

Reply:
You are right, thanks!